LiveMapperTest.java

```diff
@@ -24,5 +27,23 @@ public class LiveMapperTest {
     void 全てのliveを取得するテスト() {
         List<Live> live = liveMapper.findAll();
     }
+
+    @Test
+    @DataSet(value = "datasets/live.yml")
+    @Transactional
+    void 存在するliveのidを取得するテスト() {

```

Comment:
すみません、これまでのも含めてテストケースを以下のように見直してもらうと良いですね
```suggestion
    void 存在するliveのidを指定して取得できること() {

```

Reply:
テストケース名の変更をしました。
DBテスト側 : 17c4c9af4ce463ae374818630644638a8b1bd95d
結合テスト側 : b7fb095d7087d5b24f4c18e8d093edb0f881ebd5

Reply:
## 再テスト結果
### DBテスト結果
![スクリーンショット (170)](https://github.com/kttsu/learning_task_10/assets/150462533/582866d3-a480-4b8d-a649-33dc1750f6f7)

### 結合テスト結果
![スクリーンショット (168)](https://github.com/kttsu/learning_task_10/assets/150462533/02db6d73-1f57-4b36-91e8-265ca7463bb8)